src/integrations/prefect-docker/tests/test_worker.py

```diff
@@ -219,11 +217,14 @@ async def test_container_creation_failure_reraises_if_not_name_conflict(
         "test error"
     )
 
-    with pytest.raises(docker.errors.APIError, match="test error"):
+    with pytest.raises(ExceptionGroup) as exc:
         async with DockerWorker(work_pool_name="test") as worker:
             await worker.run(
                 flow_run=flow_run, configuration=default_docker_worker_job_configuration
             )
+    assert len(exc.value.exceptions) == 1
+    assert isinstance(exc.value.exceptions[0], docker.errors.APIError)
+    assert "test error" in str(exc.value.exceptions[0])
```

Comment:
Interesting, what's the advantage of the `ExceptionGroup` here if we're just going to make sure it's a very precise exception anyway?